spanner_orm/field.py

```diff
@@ -59,26 +74,32 @@ def __init__(self,
                field_type: Type[FieldType],
                nullable: bool = False,
                primary_key: bool = False,
-               length: int = 0):
+               length: Optional[int] = None):
+    """Initialize Field class with given parameters.
+
+    Args:
+      field_type: The column field type.
+      nullable: Whether this field should be nullable. Default is false if not provided.
+      primary_key: Whether this field should be a primary key field. Default is false if not provided.
+      length: The length of this column field type if supported. Default is MAX if not provided.
+    """
     self.name = None
     self._type = field_type
     self._nullable = nullable
     self._primary_key = primary_key
     self._length = length
 
-    if not self._type.support_length() and self._length:
-      raise error.ValidationError('length can not be set on field {}'.format(
-          self._type))
-    
-    if self._length < 0:
+    if self._length and self._length < 0:
```

Comment:
(nit) "Always use if foo is None: (or is not None) to check for a None value. E.g., when testing whether a variable or argument that defaults to None was set to some other value. The other value might be a value that’s false in a boolean context!" https://google.github.io/styleguide/pyguide.html#2144-decision (Same comment below.)